tests/accessor/accessor_exceptions_core.cpp

```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests for accessor exceptions.
+//
+// This test provides verifications that exception really has been thrown for
+// generic accessor, host_accessor and local_accessor with generic types.
```

Comment:
Looking over other PRs I've noticed that there is already some naming convention established: `local_accessor_(...).cpp`, `generic_accessor_(...).cpp` and `host_accessor_(...).cpp`

It might be reasonable to have multiple test cases within a single cpp (for example, we might have a single factory function for `types` generation). Still maybe it would be better to stick to a single naming convention. What do you think?

Reply:
for test on exception and other small tests I prefer to have one `*_core.cpp`, `*_fp16.cpp` and  `*_fp16.cpp` files
@andreyromanof @kopylovanat What do you think about this?

Reply:
Looks like it's really a not large test, as three accessors fit in here. IMO we should stick to common naming and tests separation by some criteria (accessor name as Yuri mentioned), but this solution sometimes polite too many source files that are not always necessary.
So, I'm OK with current tests separation and file naming, but we can change to common naming if you insist :)

Reply:
@andreyromanof thanks for the your opinion :)

I suggest do not change this file separation and naming
@yuriykoch What do you think?

Reply:
Fine with me, it would be easy to change if neccessary anyway)

Reply:
Ok, thanks